test/Microsoft.Extensions.SecretManager.Tests/ConfigurationExtensionTests.cs

```diff
@@ -19,37 +19,6 @@ public void AddUserSecrets_Does_Not_Fail_On_Non_Existing_File_Explicitly_Passed(
         }
 
         [Fact]
-        public void AddUserSecrets_Does_Not_Fail_On_Non_Existing_File()
```

Comment:
Please keep these. Even though we've obsoleted the API, we want to ensure we haven't regressed on the tool.